extensions/cornerstone/src/services/ViewportService/Viewport.ts

```diff
@@ -19,7 +19,7 @@ export type ViewportOptions = {
   initialView?: string;
   syncGroups?: SyncGroup[];
   initialImageOptions?: InitialImageOptions;
-  customViewportOptions?: Record<string, unknown>;
+  customViewportProps?: Record<string, unknown>;
```

Comment:
Is it a good idea to change the API surface here?  Probably this isn't used heavily enough yet to matter.